packages/web/src/components/tooltip/Tooltip.js

```diff
@@ -98,6 +108,7 @@ Tooltip.propTypes = {
 }
 
 Tooltip.defaultProps = {
+  color: themeColors['--secondary-transparent'],
```

Comment:
nice. maybe we should just move this guy to typescript and properly type/default the props